README.md

```diff
@@ -87,6 +99,33 @@ configuration Sample_xDnsServerZoneTransfer_TransferToAnyServer
 Sample_xDnsServerZoneTransfer_TransferToAnyServer -DnsZoneName 'demo.contoso.com' -TransferType 'Any'
 ```
 
+### Configuring a Primary Standalone DNS Zone
+
+```powershell
+configuration Sample_xDnsServerPrimaryZone
+{
+    param
+    (
+        [Parameter(Mandatory)]
+        [String]$ZoneName,
+        [Parameter()] [ValidateNotNullOrEmpty()]
+        [String]$ZoneFile = "$ZoneName.dns",
```

Comment:
@TravisEz13 When you create a zone, the GUI does default to `ZoneName.dns`. I will make this property optional?